tests/unit/calendar/options.js

```diff
@@ -270,46 +268,66 @@ test( "min / max", function() {
 	expect( 6 );
-	var date = new Date( 2015, 8 - 1, 1 ),
-		input = $( "#calendar" ).calendar( {
+
+	var date = new Date( 2015, 8 - 1, 1 );
+
+	// Number of month option does not work after init
+	element
+		.calendar( "destroy" )
+		.calendar( {
 			numberOfMonths: 3,
 			value: date
-		} ),
-		container = input.calendar( "widget" );
+		} );
+	widget = element.calendar( "widget" );
 
-	equal( container.find( ".ui-calendar-group" ).length, 3, "3 calendar grids" );
+	equal( widget.find( ".ui-calendar-group" ).length, 3, "3 calendar grids" );
 	equal(
-		container.find( "tbody:first td[id]:first" ).attr( "id" ),
+		widget.find( "tbody:first td[id]:first" ).attr( "id" ),
 		"calendar-2015-7-1",
 		"Correct id set for first day of first grid"
 	);
 	equal(
-		container.find( "tbody:last td[id]:last" ).attr( "id" ),
+		widget.find( "tbody:last td[id]:last" ).attr( "id" ),
 		"calendar-2015-9-31",
 		"Correct id set for last day of third grid"
 	);
 
 	// Test for jumping in weekday rendering after click on last day of last grid
-	container.find( "tbody:last td[id]:last button" ).trigger( "mousedown" );
-	equal( container.find( "thead:last th:last" ).text(), "Sa",
+	widget.find( "tbody:last td[id]:last button" ).trigger( "mousedown" );
+	equal( widget.find( "thead:last th:last" ).text(), "Sa",
 		"After mousedown last month: Last day is Saturday"
 	);
 
 	// Test if using cursor to go to the next / prev month advances three month
 	// Focus doesn't work here so we use an additional mouse down event
-	container.find( "tbody:first td[id]:first button" ).trigger( "mousedown" );
+	widget.find( "tbody:first td[id]:first button" ).trigger( "mousedown" );
 	$( document.activeElement ).simulate( "keydown", { keyCode: $.ui.keyCode.LEFT } );
-	equal( container.find( ".ui-calendar-month:first" ).text(), "May",
+	equal( widget.find( ".ui-calendar-month:first" ).text(), "May",
 		"After move to previous month: First month is May"
 	);
 
-	container.find( "tbody:last td[id]:last button" ).trigger( "mousedown" );
+	widget.find( "tbody:last td[id]:last button" ).trigger( "mousedown" );
 	$( document.activeElement ).simulate( "keydown", { keyCode: $.ui.keyCode.RIGHT } );
-	equal( container.find( ".ui-calendar-month:last" ).text(), "October",
+	equal( widget.find( ".ui-calendar-month:last" ).text(), "October",
 		"After move to next month: Last month is October"
 	);
 } );
 
+test( "value", function( assert ) {
+	expect( 4 );
+
+	var date = new Date( 2016, 5 - 1, 23 );
+
+	assert.equal( element.calendar( "option", "value" ), null, "Initial value" );
+
+	element.calendar( "option", "value", date );
+	assert.dateEqual( element.calendar( "option", "value" ), date, "Value set" );
+	equal( widget.find( "table button.ui-state-active" ).data( "timestamp" ), 1463954400000, "Active button timestamp" );
+
+	element.calendar( "option", "value", "invalid" );
+	assert.dateEqual( element.calendar( "option", "value" ), date, "Value after invalid parameter" );
```

Comment:
Perhaps we've discussed this before, but I don't remember talking through this. Is there a reason we leave the previous value instead of reverting to `null`? Native date inputs will clear their value if you try to set an invalid value.